include/stl2/view/subrange.hpp

```diff
@@ -216,6 +241,9 @@ STL2_OPEN_NAMESPACE {
 		}
 	};
 
+	template<Iterator I, Sentinel<I> S>
```

Comment:
These constraints are extraneous - the class template applies them for us.
```suggestion
	template<class I, class S>
```

Reply:
It's a combination of DRY and looking ahead to poor performance of early concepts implementations. For example, clang-concepts compiles the test suite for my libc++ implementation of `<concepts>` two orders of magnitude slower than GCC - 187 seconds vs. 1.7 seconds. This library *is* far more a research tool than a production release, so I won't complain either way.